### ISSUE TITLE
Cannot focus email more button "..." on mobile screen readers

### ISSUE THREAD

**Issue by hrb-hub** [tutao/tutanota#8183]
**Describe the bug**
When using a mobile screen reader the email "more actions" button cannot be focused

**To Reproduce**
1. In mail details view, using TalkBack keep selecting next item until you go past the mail's "more actions" button.
2. See that it does not get focused.

**Expected behavior**
We should be able to focus the mail's "more actions" button on mobile screen readers.

**Screenshots**
![Image](https://github.com/user-attachments/assets/430b25d4-f97d-4fdc-b310-d3f7508c06b8)

## Test notes
- [ ] Using TalkBack
  - [ ] "more actions" button can be focused and clicked
  - [ ] Mail sender text can be:
    - [ ] Focused
    - [ ] Clicked to expand and collapse mail
- [ ] Tabbing using a keyboard
  - [ ] "more actions" button can be focused and clicked
  - [ ] Mail sender text can be:
    - [ ] Focused
    - [ ] Clicked to expand and collapse mail
